src/Xamarin.Android.Build.Tasks/Tasks/CreateAar.cs

```diff
@@ -102,6 +104,15 @@ public override bool RunTask ()
 						existingEntries.Remove (archivePath);
 					}
 				}
+				if (ProguardConfigurationFiles != null) {
+					var sb = new StringBuilder ();
+					foreach (var file in ProguardConfigurationFiles) {
+						string[] lines = File.ReadAllLines (file.ItemSpec);
```

Comment:
Could this use `File.OpenText()` and `ReadLine()` instead of creating a `string[]`? Then it could add directly to the `StringBuilder`.